static/app/components/bulkController/bulkNotice.tsx

```diff
@@ -17,22 +17,14 @@ function getSelectAllText(allRowsCount?: number, bulkLimit?: number) {
 
   if (bulkLimit && allRowsCount > bulkLimit) {
     return {
-      noticeText: tct('Selected up to the first [count] items.', {
```

Comment:
this change just renders fewer spans

Reply:
I remember @matejminar liked to use `tct` because it would make things easier for a person translating it to other languages

Reply:
yeah, it definitely makes sense when we use more than one variable, in this case maybe less so 🤷‍♂️